### DISCUSSION TITLE
잠시후 시도하세요가 많이나오면 자동으로 한시간 멈추고 다시 작동하는건 어떨까요

### DISCUSSION THREAD

**Discussion by swoo9799** [changdoc/naver-vaccine-macro#169] [Ideas]
잠시후 시도하세요가 많이나오면 자동으로 한시간 멈추고 다시 작동하는 기능을 on off 하는건 어떨까요?

**Comment by changdoc** [comment]
잠시 후 시도하세요가 나왔다는 건 네이버에서 차단하는 기준으로 새로고침을 시도 했다는 건데요.
한시간 후에 다시 시도 하도록 변경해도 다시 차단될 가능성이 높을거 같은데요.
매크로 작동 하는 쪽에서 차단 안당하도록 동작 시키는게 맞는 방향 같네요.

**Reply by hosun9lee** [reply]
이미 차단당한 상태여서 '잠시 후 다시 시도하세요'가 뜨면 어떻게 해야 다시 시도할 수 있을까요?
'잠시 후 다시 시도하세요'가 뜨더라도 한시간 후면 풀려서 다시 매크로 돌릴 수 있을까요?

**Reply by changdoc** [reply]
네 한시간 후에 신청 페이지 접근해보시면 다시 동작 가능하실거에요.
